### PR TITLE
S-EL1 SPMC fixes

### PR DESCRIPTION
**Pull request by balint-dobszay-arm**

### CODE REVIEW

File: core/kernel/tpm.c [hunk 1]
```diff
@@ -47,12 +47,14 @@ static int read_dt_tpm_log_info(void *fdt, int node, paddr_t *buf,
 
 	log_addr = fdt32_to_cpu(property[1]);
 
+#ifndef CFG_CORE_SEL1_SPMC
```
Comment (jenswi-linaro) [inline core/kernel/tpm.c:50]: Please use `if (!IS_ENABLED(CFG_CORE_SEL1_SPMC))` instead.

**Review by jenswi-linaro — COMMENTED** [review]
With my comment addressed, please apply:
`Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: core/kernel/tpm.c (hunk 1 above)
Comment (balint-dobszay-arm) [reply core/kernel/tpm.c]: Fixed

**Comment by jenswi-linaro** [comment]
Looks good.

**Pull request merged**